### DISCUSSION TITLE
Top max pts events a la semaine (+ badge ?)

### DISCUSSION THREAD

**Discussion by Bombafox** [Crownicles/Crownicles#861] [Nouvelles idées]
Vu qu'on en parle dans le général en ce moment j'avais aussi pensé à un top des gens qui on touché beaucoup de pts en 1 event

Soit comme le top donc ça s'actualise que si le joueur dépasse son ancien record ou alors plutôt comme un tw avec un badge

**Comment by Pietagorh** [comment]
Peut-être plutôt le rajouter au minievent de stats ?
